gwpy/cli/cliproduct.py

```diff
@@ -888,6 +890,11 @@ def _finalize_arguments(self, args):
             args.xmin = min(starts)
         if args.epoch is None:
             args.epoch = args.xmin
+        else:
+            epoch = float(args.epoch)
```

Comment:
@areeda, why is `args.epoch` cast to a new variable here? why not just

```python
        if args.epoch is None:
            args.epoch = args.xmin
        elif args.epoch < 1e8:  # user gave epoch relative to start
            args.epoch += min(starts)
```

Reply:
Does that work with LIGOTimeGPS? I thought it was necessary.

Reply:
Yes, that should work, `LIGOTimeGPS` is compatible with floats.